arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -0,0 +1,1580 @@
+        }
+      },
+      "description": "Azure Storage blob container properties information."
+    },
+    "BlobContainer": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the name of the blob container."
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the unique identifier of the blob container."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the type of the blob container."
+        },
+        "properties": {
+          "readOnly": true,
+          "$ref": "#/definitions/BlobContainerProperties",
+          "description": "the properties of the blob container."
+        }
+      },
+      "description": "Azure Storage blob container information."
+    },
+    "ListBlobContainersResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BlobContainer"
+          },
+          "readOnly": true,
+          "description": "the results of the list operation"
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "the link (url) to the next page of results."
+        }
+      },
+      "description": "The list of blob containers associated with the storage account attached to the Data Lake Analytics account."
+    },
+    "SasTokenInfo": {
+      "properties": {
+        "accessToken": {
+          "readOnly": true,
+          "type": "string",
+          "description": "the access token for the associated Azure Storage Container."
+        }
+      },
+      "description": "SAS token information."
+    },
+    "ListSasTokensResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SasTokenInfo"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the link (url) to the next page of results."
+        }
+      },
+      "description": "The SAS response that contains the storage account, container and associated SAS token for connection use."
+    },
+    "DataLakeStoreAccountInfoProperties": {
+      "properties": {
+        "suffix": {
+          "type": "string",
+          "description": "the optional suffix for the Data Lake Store account."
+        }
+      },
+      "description": "Data Lake Store account properties information."
+    },
+    "DataLakeStoreAccountInfo": {
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "the account name of the Data Lake Store account."
+        },
+        "properties": {
+          "$ref": "#/definitions/DataLakeStoreAccountInfoProperties",
```

Comment:
added everywhere except for JobProperties, which as we discussed can't use it since it has a discriminator and that would be strange.